src/app/components/article/MediaArticleCard.js

```diff
@@ -117,6 +124,7 @@ MediaArticleCard.propTypes = {
   publishedAt: PropTypes.number, // Timestamp
   variant: PropTypes.oneOf(['explainer', 'fact-check']),
   onClick: PropTypes.func,
+  id: PropTypes.number.isRequired,
```

Comment:
Question: is this really a number or is it a string?

Reply:
Oh shoot, you're right, fixed

Reply:
Awesome, thanks!